recipes/python-symengine/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.1.0.git" %}
+{% set commit = "4361e3782ce2b61c9962f3f9a1e766e481a00bf9" %}
+
+package:
+  name: python-symengine
+  version: {{ version }}
+
+source:
+  fn: {{ commit }}.tar.gz
+  url: https://github.com/symengine/symengine.py/archive/{{ commit }}.tar.gz
+  sha256: a28803ffbb15d4b252dad18d252d79419d1d561c1ead9ad1671821527c1f94a2
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - symengine
+    - python
+    - cython 0.23*
```

Comment:
One is https://mail.python.org/pipermail/cython-devel/2016-April/004771.html
There are some more errors when I remove the offending lines from the cython file.